### PR TITLE
backend tests: devauth tests cleanup

### PR DESCRIPTION
**Pull request by kjaskiewiczz**

### CODE REVIEW

File: tests/tests/test_os_ent_migration.py [hunk 1]
```diff
@@ -21,7 +21,7 @@
 from testutils.api.client import ApiClient
 from testutils.infra.cli import CliTenantadm
 import testutils.api.deviceauth as deviceauth_v1
-import testutils.api.deviceauth_v2 as deviceauth_v2
+import testutils.api.devicedevauth as devicedevauth
```
Comment (lluiscampos) [inline tests/tests/test_os_ent_migration.py]: Where is this `devicedevauth` coming from?

Comment (lluiscampos) [reply tests/tests/test_os_ent_migration.py]: I see you removed it in a later commit, so probably just a temporary hack for search+replace later...

File: backend-tests/tests/test_deployments.py [hunk 2]
```diff
@@ -10,8 +10,7 @@
 from datetime import datetime, timedelta
```
Comment (lluiscampos) [inline backend-tests/tests/test_deployments.py:10]: Once we are on it: please add the Copyright notice here

File: backend-tests/tests/test_rbac.py [hunk 3]
```diff
@@ -5,7 +5,6 @@
```
Comment (lluiscampos) [inline backend-tests/tests/test_rbac.py:5]: And here

File: backend-tests/tests/test_tenantadm.py [hunk 4]
```diff
@@ -5,10 +5,14 @@
```
Comment (lluiscampos) [inline backend-tests/tests/test_tenantadm.py:5]: And here :)

File: backend-tests/tests/test_tenantadm_v2.py [hunk 5]
```diff
@@ -8,8 +8,14 @@
```
Comment (lluiscampos) [inline backend-tests/tests/test_tenantadm_v2.py:8]: Here too

File: backend-tests/tests/test_devauth.py [hunk 6]
```diff
@@ -12,22 +12,1401 @@
...
+        dev.authsets.append(new_set)
+
+    dev.status = "pending"
+
+    return dev
+
+
+def make_accepted_device_with_multiple_authsets(
+    utoken, num_auth_sets=1, num_accepted=1, tenant_token=""
+):
```
Comment (lluiscampos) [inline backend-tests/tests/test_devauth.py:405]: Just curious, but what is the motivation for renaming this helper? The other ones (`make_pending_device`, for instance) also seem to work with multiple authsets.

Comment (mchalski) [reply backend-tests/tests/test_devauth.py:405]: +1

File: tests/tests/test_os_ent_migration.py (hunk 1 above)
Comment (kjaskiewiczz) [reply tests/tests/test_os_ent_migration.py]: I didn't remove it, I've combined _deviceauth.py_ and _deviceauth_v2.py_ into one _deviceauth.py_ file in testutils/api/

File: backend-tests/tests/test_devauth.py (hunk 6 above)
Comment (kjaskiewiczz) [reply backend-tests/tests/test_devauth.py:405]: there is already `make_accepted_device` method in the testutils.common and the tests imports it so I had a method name conflict

Comment (kjaskiewiczz) [reply backend-tests/tests/test_devauth.py:405]: and with that method, it's not possible to create a device with multiple authsets

File: tests/tests/test_os_ent_migration.py (hunk 1 above)
Comment (lluiscampos) [reply tests/tests/test_os_ent_migration.py]: My point is that `devicedevauth` does did not exist, nor it existed at any point on the Git history. You probably replaced "auth_v2" with "devauth" and hence ended up with this "devicedevauth" here.

**Pull request merged**